lib/zluxArgs.js

```diff
@@ -107,6 +108,17 @@ if(process.env.overrideFileConfig !== "false"){
     }
     configJSON = mergeUtils.deepAssign(configJSON, envConfig);
   }
+  let zoweDotYamlConfig;
+  const zoweDotYamlFile = yamlConfig.getDefaultZoweDotYamlFile();
+  const haInstanceId = yamlConfig.getCurrentHaInstanceId();
+  if (zoweDotYamlFile && haInstanceId) {
+    const componentOrder = ['zss', 'app-server']; // from lower to higher priority
+    zoweDotYamlConfig = yamlConfig.getConfig(zoweDotYamlFile, haInstanceId, componentOrder);
+  }
+  if (cluster.isMaster && zoweDotYamlConfig) {
+    console.log(`\nZWED5019I - Processed zowe.yaml config\n${JSON.stringify(zoweDotYamlConfig, null, 2)}`);
+  }
+  configJSON = mergeUtils.deepAssign(configJSON, zoweDotYamlConfig);
```

Comment:
This will crash if zoweDotYamlConfig is undefined, which is true when both zowe.yaml and instance.env exist.

Reply:
Fixed